src/endpoint.h

```diff
@@ -58,7 +58,7 @@ struct TcpEndpointConfig {
     std::string name;
     std::string address;
     unsigned long port;
-    int retry_timeout;
+    int retry_timeout{5};
```

Comment:
Commenting here, but this is about the commit message:

```
Previously some default values were set in the struct definition and others
explicitly in the parsing functions. Not there's just one place: In each config
struct's definition.
```

Typo: "Now there's ..."